### ISSUE TITLE
How can I set no. of list batch?

### ISSUE THREAD

**Issue by mactrash** [bsuttor/blog.post#2]
@bsuttor 

I cannot find it, can you help?

**Comment by bsuttor** [comment]
Hi,
I'm sorry but I don't understand the question?

I make this package for my personal blog, and unfortunately, I have not time to work on it at this moment.

But PR are welcomed if you want to improve this package ;)

Regards,

**Comment by mactrash** [comment]
@bsuttor 

I mean is the any where can set the batch size of the blogs_view list / render? i.e. list 8 blog post per page.

**Comment by bsuttor** [comment]
Indeed, I didn't think of that when I created this package, It should be added.
I think best way to do that is adding a registry record and use it with a batch for the 'blogs_view'

http://docs.plone.org/external/plone.app.contentlisting/README.html
